### PR TITLE
Made URLs more RESTful.

### PR DESCRIPTION
**Pull request by Vlad-Enia**

### CODE REVIEW

File: src/main/java/ro/uaic/info/taskhandler/controller/AnswerController.java [hunk 1]
```diff
@@ -44,13 +45,12 @@
             questionId = Integer.parseInt(answer.get("questionId"));
             taskId = Integer.parseInt(answer.get("taskId"));
             content = answer.get("content");
-        }
-        catch (Exception e)
+        } catch (Exception e)
```
Comment (robertadriang) [inline src/main/java/ro/uaic/info/taskhandler/controller/AnswerController.java:48]: try{
} catch {
} 
OR
try{
}
catch
{
}

File: src/main/java/ro/uaic/info/taskhandler/controller/AnswerController.java [hunk 2]
```diff
@@ -141,20 +127,16 @@
...
```
Comment (robertadriang) [inline src/main/java/ro/uaic/info/taskhandler/controller/AnswerController.java:130]: try{
} catch {
}
OR
try{
}
catch
{
}

File: src/main/java/ro/uaic/info/taskhandler/controller/ScoreAnswerController.java [hunk 3]
```diff
@@ -44,15 +45,11 @@
             questionId = Integer.parseInt(scoreAnswer.get("questionId"));
             taskId = Integer.parseInt(scoreAnswer.get("taskId"));
             scoreValue = Integer.parseInt(scoreAnswer.get("scoreValue"));
-        }
-        catch (Exception e)
+        } catch (Exception e)
```
Comment (robertadriang) [inline src/main/java/ro/uaic/info/taskhandler/controller/ScoreAnswerController.java:48]: try{
} catch {
}
OR
try{
}
catch
{
}

File: src/main/java/ro/uaic/info/taskhandler/controller/ScoreAnswerController.java [hunk 4]
```diff
@@ -141,20 +122,15 @@
...
```
Comment (robertadriang) [inline src/main/java/ro/uaic/info/taskhandler/controller/ScoreAnswerController.java:125]: try{
} catch {
}
OR
try{
}
catch
{
}

File: src/main/java/ro/uaic/info/taskhandler/controller/TaskStudentController.java [hunk 5]
```diff
@@ -66,91 +60,92 @@
...
             found.get().setStatus("started");
-            TaskRegistration updatedTask= taskRegisterRepository.save(found.get());
-            if(updatedTask==null)
-                return ResponseEntity.notFound().build();
+            TaskRegistration updatedTask = taskRegisterRepository.save(found.get());
             return ResponseEntity.ok(updatedTask);
-        }
-        else
-            return  ResponseEntity.badRequest().build();
+        } else
```
Comment (robertadriang) [inline src/main/java/ro/uaic/info/taskhandler/controller/TaskStudentController.java:126]: }
else
}
OR
}else{

**Review by robertadriang — CHANGES_REQUESTED** [review]
Please reformat the code :')

File: src/main/java/ro/uaic/info/taskhandler/controller/ScoreAnswerController.java (hunk 4 above)
Comment (vladvlad00) [reply src/main/java/ro/uaic/info/taskhandler/controller/ScoreAnswerController.java:125]: nu

**Review by vladvlad00 — APPROVED** [review]

**Review by robertadriang — APPROVED** [review]

**Pull request merged**